### PR TITLE
Event listener

### PR DESCRIPTION
**Pull request by mario-mitchell**
Fixed conflict issues. Submit button logging info requested in the console.

### CODE REVIEW

File: javascript-job-search-mvp/index.html [hunk 1]
```diff
@@ -75,7 +75,11 @@
 						<li><a href="https://vetswhocode.io/apply"><span class="link-name">Apply</span></a></li>
 						<li><a href="https://vetswhocode.io/contact"><span class="link-name">Contact</span></a></li>
 						<li><a href="https://vetswhocode.io/blog"><span class="link-name">Blog</span></a></li>
+<<<<<<< HEAD
```
Comment (dtetreau251) [inline javascript-job-search-mvp/index.html:78]: this was cut and pasted from your last PR. the <<<< HEAD doesn't need to be there, the ====== doesn't need to be there, and the  >>>>> main doesn't need to be there.

Comment (dtetreau251) [inline javascript-job-search-mvp/index.html:78]: a little worried about these markers. I think they may have been pasted with your code when you redid your PR.

**Comment by joeromine** [comment]
Mario, are you seeing the merge conflicts on this branch?

File: javascript-job-search-mvp/index.html (hunk 1 above)
Comment (dtetreau251) [reply javascript-job-search-mvp/index.html:78]:
```
// All variables for Submit Button 
const submitButton = document.getElementById("submit-button");
const zipCode = document.getElementById("zipCode");
const isRemote = document.getElementById("inlineFormCheck");
const range = document.getElementById("range");
const formData = {};
// Triggers the function when the search button is clicked
function submitButtonEvent (event) {
  event.preventDefault()
  formData.zipCode = zipCode.value;
  formData.isRemote = isRemote.checked;
  formData.range = range.value;
  console.log(formData);
}
submitButton.addEventListener("click", submitButtonEvent);
```
I think what went wrong is that you pasted the code from your last pull request to try to fix the merge conflict. You inadvertently pasted the markers it displayed in the last merge conflict. I think what I would do is:
1. rebase main on your machine
2. make a brand new branch with a different name than you used before
3. paste the code above in the javascript file
4. push the code and do another pull request

Either do that or go through your code on this branch on your computer and remove all the markers.

**Comment by mario-mitchell** [comment]
I'm seeing no merge conflicts. I actually fixed all the conflicts without having to copy and paste my code over. I just had to get it up to date. I have no issues.

**Comment by dtetreau251** [comment]
> I'm seeing no merge conflicts. I actually fixed all the conflicts without having to copy and paste my code over. I just had to get it up to date. I have no issues.

Mario, look in the files changed at the top.

**Comment by mario-mitchell** [comment]
Fixing the HTML.

**Pull request closed without merging**